### DISCUSSION TITLE
Icon beta feedback collection

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#11912] [General]
I wanted to start this to paste some feedback we received for the beta version (positive and negative). I will leave out posts which don`t provide any constructive feedback (such as "I dont like them) but will instead ask for a more detailed feedback and add it here once I receive something.

I will post each new feeedback as a new post, so members can us the reply to post feature to comment individual feedbacks.
@cgeo/team If you see any feedback out there, feel free to post it here as well.

This is just to get an impression of the feedback we received and is neither complete nor representative. So mainly this issue is meant to deliver some impulses or might also not provide anything useful...we will see.

I will open up another discussion to think about a survey (in some minutes).

**Comment by Lineflyer** [comment]
Google Play Console:

Last cache icon update is crap. Earthcache not recognizable. Visited stages not transparent.

**Reply by ztNFny** [reply]
"crap" - i love productive feedback ...

**Reply by Lineflyer** [reply]
Yeah...don't count the first sentence...I pasted it here only because of the second half.

**Comment by Lineflyer** [comment]
Google Play Console:

Die neuen Symbole sind leider eher verwirrend weil sie sehr stark von den Alten und somit von Geocaching.com abweichen. Farblich als auch von der Struktur. Schade, leider ein Schritt rückwärts,...

**Reply by Lineflyer** [reply]
IMHO they got closer to GS icons as ever before...

**Reply by ztNFny** [reply]
Don't understand the feedback about being more different from GC 🤨

**Comment by Lineflyer** [comment]
Google Play Console:

I give you my opinion about the new icons. They are very difficult to see. We need to zoom in a lot to distinguish one cache from an other. Earthcaches look like a Wi-Fi sign and it took me a while to recognize the others except trad of course. There is not enough contrast to see them easily. Minwhile it's really Nice to be able to see what was the last logs on the main page.

**Comment by Lineflyer** [comment]
From support mail:

Well done with those new icons. Looks really modern and structured now. Especially the different colors seem logical now.

**Comment by Lineflyer** [comment]
From support mail:

I need to get used to the icons. But some are really strange. Whats the meaning of multi cache icon, but at least I can remember the color. The earth cache icon however looks like a Wifi signal and I can't get used to that one.

**Comment by Lineflyer** [comment]
From personal mail:

Vielen Dank für die beständige Weiterentwicklung. Die neuen Symbole sind super. Die Umkehr der Farben (z.B. grünes Icon statt weißes Icon mit grünem Symbol) machen das Ganze viel strukturierter. Ich muss mich aber noch an einige gewöhnen (Multi, Event und Earthcache unterscheiden sich schon sehr stark vom vorherigen und vom gewohnten aus anderen Plattformen...da könnte man evtl. noch optimieren).

**Reply by ztNFny** [reply]
Event is a "speech bubble" on GC as well?? Could you ask what he means?

**Reply by Lineflyer** [reply]
Will do

**Reply by Lineflyer** [reply]
The user was looking at a community celebration event, which has a different icon as "normal" events.
Now that he learned that, all is fine for him with respect to events and he even likes, that special events have a different icon.

**Comment by Lineflyer** [comment]
From Facebook:

hmmm ich persönlich finde die Überarbeitung der Icons mehr als entbehrlich, vor allem dass die Farben jetzt nur noch kitschig sind und ein Multi jetzt komplett anders aussieht. Violett für die Labs ist auch entbehrlich, das rot hat schon gut gepasst. Weniger ist oft viel besser als mehr!

**Comment by Lineflyer** [comment]
From Facebook:

Just got the RC update with the new icons.  So sexy!  Y'all are on fire these days! Thanks for all the great work you guys do!

**Comment by Lineflyer** [comment]
From Facebook:

Ich finde sie unschön, fand die alten Cachesymbole besser, sie waren farblich unterschiedlicher und somit bei starker Sonneneinstrahlung besser zu unterscheiden.
Auch die zusätzlichen Symbole die am Cachesymbol hängen finde ich in farblicher Abänderung gewöhnungsbedürftig, z.b.: gespeichert, Notiz, gelöster Wimpel, gerade letzterer ist nicht mehr so prägnant zu erkennen wie vorher,... Früher: Rotes Fähnchen an blauem Mystery, bzw Weißer Letterbox, war besser zu erkennen als nun nun ein blaues Fähnchen an blauem Mystery.
Das aus technischer Sicht andere (waren es Vektor-?) Grafiken Nötig sind ist ja ok, wünschenswert wäre mir trotzdem die vorherige Darstellung.

**Reply by Lineflyer** [reply]
The point I agree to is, that the red final marker was better (crying out loud: "This has corrected coords...go get it").

**Reply by ztNFny** [reply]
Fyi, red flag marker is now used for "has final waypoint" (which was using the same icon before, so a new one was needed). Blue marker is what GC is using for "correctee coords", that's why I picked it to be aligned.
We already had a long discussion about these 2 markers, coming to the current ones.

**Reply by Lineflyer** [reply]
I remember the discussion.
But be aware (maybe you did not know): The old flag marker was also set for "corrected coords", so users with that workflow have seen it a lot on their solved mysteries and will now see something else.

But AFAIR there was a majority for the light blue one...so I have to accept it.

**Reply by ztNFny** [reply]
Yes, the flag was used (wrongly) for 2 different cases in the past. To resolve that one of these cases needs to change.

**Reply by Lineflyer** [reply]
And I proposed it the other way around in order to keep the (IMHO) common use case of corrected coords with an unchanged icon.
But I don't want to start the discussion over again, just posting in case I was misunderstood during the original discussion.

**Comment by Lineflyer** [comment]
From Facebook:

 Die Cache Symbole sind soweit ganz gut das die Farben bei behalten wurden!

**Comment by Lineflyer** [comment]
From Facebook:

 Ich fand die ursprünglichen Symbole voll ok. Sie sind die gleichen wie die gebräuchlich bei GC. Jetzt musste ich mich umgewöhnen. Ich habe "benutze kompakte Symbole" ausgeschaltet, da sie beim Zoomen nicht mit skaliert werden, was ich doof finde.
Ich weiß es zu schätzen, dass ihr ständig was verbessern möchtet und dass ihr es gut meint. Aber denkt mal auch sn die Menschen, die Schwierigkeiten haben, sich an neues zu gewöhnen, die Probleme habe sich neue Symbole zu merken. Es sind nicht alle Cacher Einsteins oder IT Spezialisten.

**Reply by Lineflyer** [reply]
My answer:

Danke fürs Feedback. Sind die neuen Symbole (insbesondere in ihrer Farbgebung) nicht sogar nun ähnlicher wie bei GC?
![image](https://user-images.githubusercontent.com/949669/137588814-a417ff23-c3b7-4331-bc9b-d47f26f60172.png)

**Reply by Lineflyer** [reply]
Answer by user:
Farbe ja, die orangen Doppeldosen beim Multi und die Erde beim Earthcache fand ich aber besser. Dass die Achtecke persönliche Wegpunkte sind musste ich dann erst wissen. Ich suchte diese am Anfang wie verrückt. Das mit dem Zoom hab ich ja erwähnt, jetzt ist es besser erkennbar, wenn ich herrauszoome

**Reply by ztNFny** [reply]
I wonder why people who don't like changes are using a beta though?

**Reply by ztNFny** [reply]
I dont understand the comments regarding zoom?

**Reply by Lineflyer** [reply]
Maybe I cutted it of as it was unrelated. It was something about that compact icons automatic does not zoom but switch.

**Comment by Lineflyer** [comment]
Facebook:

Die Symbole sehen sowei gut aus! Am was ich mich erst gewöhnen muss, sind die benutzerdefinierten Ausgangspunkte! Da fnd ich die Altem besser....
Grund die waren einfacher ersichtlich! Also etwas deutlicher wäre nicht falsch

![image](https://user-images.githubusercontent.com/949669/137600936-b8e71665-43eb-4195-8a8e-f621b8fbbdb5.png)

**Reply by ztNFny** [reply]
![Screenshot_20211016-221211](https://user-images.githubusercontent.com/1258173/137600995-8fbf5fbc-2fa6-4ae9-96b0-3caef1f7706e.jpg)
Icon and color stayed mostly the same?

**Reply by Lineflyer** [reply]
Wheren't those dark brown before...not sure though.

**Reply by ztNFny** [reply]
Attached screenshot. Looking at the old one, that one is very hard to recognize as a sign imho ...

**Reply by ztNFny** [reply]
Maybe what he means is the size? As discussed I'll play around with the sizes again a bit.
Could you ask him to detail his feedback?

**Reply by Lineflyer** [reply]
Can try...need to find the post tomorrow.

**Reply by Lineflyer** [reply]
Question asked:

Die alten Ausgangspunkte waren eigentlich schwerer zu erkennen, oder? Siehe Bild. Links fehlt nur die Blase drumrum auf diesem Bild.
Oder meinst du vielleicht die Größe innerhalb der Blase? Da arbeiten wir noch daran, sie ggf. etwas größer zu machen.

**Reply by ztNFny** [reply]
funny thing, the size of those icons is already larger than I intended as I forgot to change a 24 to 22 when creating the drawables from svg 😄

**Comment by Lineflyer** [comment]
Facebook:

Am Anfang fand ich die neuen nicht so gut. Hab mich aber jetzt daran gewöhnt und finde sie gut.
Nur das Wig und EC müsste deutlicher unterschieden sein.

**Comment by Lineflyer** [comment]
Facebook: 

Man muss sich kurz gewöhnen, aber dann ist okay. Wie meistens.

**Comment by Lineflyer** [comment]
Facebook: 

Im ersten Moment fand ich es sehr unübersichtlich. Das Symbol für Multis finde ich passender als die vorherigen 3 Dosen. Das Symbol für den Earthcache finde ich aber nicht gut. Es geht in der Karte, bei den ganzen blauen (Wherigo, Letter, Mystery) etwas unter und farblich hätte ich Braun (Erde) besser gefunden. Vielleicht könnte man die Wherigos, Mysterys und Letterboxen noch etwas unterschiedlicher machen. Gut finde ich die Smileys für die letzten Funde.

**Reply by ztNFny** [reply]
Finally someone mentioning multi in a positive way 😄
Most multis don't have multiple boxes, so I find the "waypoints" a lot more representable of reality.

**Comment by Lineflyer** [comment]
Facebook:

Mir persönlich haben die alten Symbole besser gefallen. Nach über 10 Jahren cgeo Nutzung ist das schon eine Umstellung. Insgesamt finde ich es sehr "blaulastig" (Webcam, Virtual, Wherigo, Earthcache) Auf einer gut gefüllten Karte waren das vorher immer gute Orientierungshilfen. Das bunte EC Symbol, der gelbe Bleistift bei Notizen und das rote Fähnchen bei eingetragenen Finalkoords waren einfach realistischer. Die dunkleren Schuhe bei den Wegpunkten sind auch weniger auffällig, gerade beim dunklen Design. Ich finde ehrlich gesagt diese Designänderung eher verschlimmbessert und mir erschließt sich nie der Grund für derlei Änderungen. 🤔
Aber prinzipiell wäre auch nix dagegen zu sagen, wenn man es optional anpassen kann. 🙏
Wir haben in unserer Regionalgruppe eine Umfrage gemacht (bei 65 abgegebenen Stimmen).
Nichtsdestotrotz macht ihr sonst eine Top Arbeit 👍😎

![image](https://user-images.githubusercontent.com/949669/137601269-b7e0b2f5-e2b6-48bd-b811-5db5b10aec62.png)

**Reply by ztNFny** [reply]
The missing marker background in waypoint list is a known issue.

**Comment by Lineflyer** [comment]
Facebook:

Für mich ist das Multi Logo nur eindeutig wegen der Farbe. Die Symbole sind nicht klar. Der Earthcach ist unpassend und der Wig wäre klarer wenn es nur den Pfeil ohne den Kreis geben würde.
Ansonsten eine gelungene Auffrischung der Logos!

**Comment by Lineflyer** [comment]
Facebook:

Ich finde das neue Symbol für Multi auch nicht gut. Das klassische in Gelb war eindeutig und entsprach dem von Groundspeak. Jetzt sieht's aus wie ein E-Scooter. 😉
Grundsätzlich bin ich immer für Modernisierung, aber da eh schon so viel neu ist (Filter etc.), muss man immer noch sehr genau schauen und jetzt mit neuen Symbolen leider noch mehr.

**Comment by Lineflyer** [comment]
Facebook:

finde die Symbole sehr schön anzusehen, würde nur einen größeren Farbunterschied zw. Mysterie und Adv.Lab. begrüßen. Die sind bei bei weiter ausgezoomter Karte schlecht zu differenzieren.

**Reply by Lineflyer** [reply]
Blue and purple...its totally different already IMHO.

**Comment by Lineflyer** [comment]
Support mail:

Nice new beta update. Much more structured icons on the map. Like it despite Multi and Earth, which somehow don't transfer their meaning.

**Comment by Lineflyer** [comment]
Facebook (c:geo page):

I think you changed the symbol for the earthcache now, but i still think it should be a kind of brown

**Comment by Lineflyer** [comment]
Facebook (c:geo page):

I love the new icons they're so much easier to see.

**Comment by Lineflyer** [comment]
Facebook (c:geo page):

I like the item set, the items are very clear in their symbol and colour. With one exception: the EarthCache icon looks more like a wifi logo now. The non-earth-ish blue colour contributes to that. The previous multi-coloured icon inspired by the original EarthCache logo was easier to spot.

**Comment by Lineflyer** [comment]
Facebook (c:geo page):

In general i'm very pleased with the Change. The only icon I don't like is the multicache. I know that there was another option, that I liked 🙂

**Comment by Lineflyer** [comment]
Facebook (c:geo page):

A fresher look without looking too "modern art".
I love c:geo because it's sleek and clear. As long as it's doesn't interfere with functionality and speed, I'm ok with it.

**Comment by ztNFny** [comment]
I think at this point we can kinda stop collecting the feedback, at least as long as it doesn't bring up any new points (to save @Lineflyer some work)

So to summarize:

> Too much is changing (not just icons but new filter, UI overhaul, ...)

That's a "problem" every app has - nothing can be done about this (other than deciding to never ever again change something, but I'm pretty sure we don't want to go that route).
Wait for the feedback when they see bottom nav 😉

> Earthcache icon

Already on the todo list

> Multi icon

Well - *I* like the new icon a lot more than the old 2 boxes.

Reasons:
- Waypoints represent a multi better than multiple boxes. Every multi has waypoints, only some have boxes/physical stages.
- Doing a "multi box" icon is quite hard. Imho the one used by GC looks badly, in as that it's hard to recognize the icon at small size, there's just too much content in that icon, causing everything to be small and detailed and that scales badly.
Below is a screenshot from GC.coms icon page, resized to 20%. For most icons you can make out the basic shape, but for multi it's hard to recognize - unless you already know what it's supposed to be - but "having to know it" is bad design imho. The light background (orange being one of the lightest colors in the palette) and thus the low contrast don't help that either.
![Screenshot gc](https://user-images.githubusercontent.com/1258173/137621670-7ad1cf99-f39b-460f-8995-e484ca01d948.png)

Comparing them in large it's ok (original GC icon on the right):
![image](https://user-images.githubusercontent.com/1258173/137622139-7449347b-06a4-4c6d-963a-563aed0d3a7b.png)

But zoom that out and the GC icon becomes unidentifyable:
![a](https://user-images.githubusercontent.com/1258173/137622191-d21472dc-931e-4e36-8b22-84c5fdd4bfd6.png)

If it's really wanted by the team I'll create a 2 box icon (I'll just stay using mine in my builds 😜), but I have no idea how to solve the problems mentioned above (and neither does GC who are paying designers a lot of money, so it's probably unsolvable), but maybe it's "better" just because it's legacy?

Another Option: We give other waypoint icon I had at the beggining a try? (with 4 waypoints instead of 3)

@cgeo/team Could you please voice your feedback regarding multi icon here? I'd suggest to use the reaction-smileys as a kind of vote function: Upvote = stay with current, Downvote = create 2 boxes design, Smiley = Use the other waypoints icon

**Reply by Lineflyer** [reply]
Voted smile because I think the 4 stage "following a path" icon could be a good compromise and still be distinguishable on low zoom.

Regarding collection of reactions:
I will keep posting some here, but then only those bringing a new proposal / point.

Regarding the "too much changes":
Well yes, there was a lot of change and not every user can follow up so quickly with new looks, etc. For occasional cachers it might even look different each time they use the app ;)  
However in some months most will get used to it. Fun fact: I showed a 2020 version, which happend to be installed on one of my old devices to a friend and he said "Was it really that ugly before?" :)

**Reply by ztNFny** [reply]
Noone else have an opinion on that?

**Reply by Lineflyer** [reply]
@cgeo/team Please help @ztNFny with a vote as described above. That would be helpful.

**Reply by eddiemuc** [reply]
I am really fine with any of the three. Thus I voted the "party" head as "neutral"

**Reply by murggel** [reply]
For me both path-variants are fine, I can go without the two boxes.
So one thumb up and one smiley from my side.

**Reply by ztNFny** [reply]
> For me both path-variants are fine, I can go without the two boxes. So one thumb up and one smiley from my side.

Strange, there's no thumbs up vote?

**Reply by bekuno** [reply]
> Strange, there's no thumbs up vote?

Instead there are 4 upvote votes: ![grafik](https://user-images.githubusercontent.com/32555742/137914418-61b89f55-d2d4-40e9-bc06-2bcd5c472b74.png)

**Reply by bekuno** [reply]
I like the "way" icon. How about combine the way with a single box?

**Reply by ztNFny** [reply]
> > Strange, there's no thumbs up vote?
> 
> Instead there are 4 upvote votes:

If the upvotes are "thumbs ups" then it seems that the current icon is liked best - and noone is voting for the classic 2 boxes.
Edit: ah crap, I just noticed i wrote "Upvote", not thumbs up. So the current seems to be favored internally.

Suggestion:
1) Beta 2 with the other (4-point) waypoint icon
2) After beta 2 do a survey with the 2 waypoint icons and go with the majority

**Reply by ztNFny** [reply]
> I like the "way" icon. How about combine the way with a single box?

Same problem as 2 boxes: Too many details, will get too small.

**Reply by Lineflyer** [reply]
> Suggestion: [...]

Sounds good!

**Reply by bekuno** [reply]
> 
> 
> > I like the "way" icon. How about combine the way with a single box?
> 
> Same problem as 2 boxes: Too many details, will get too small.

And how about a combination of the 3 and 4 point icon ->  3 points, but use the final marker from the 4 point icon ?

**Reply by ztNFny** [reply]
> And how about a combination of the 3 and 4 point icon -> 3 points, but use the final marker from the 4 point icon ?

Do you think not recognizing the icon as "waypoints" is the problem people are having? From the comments I've got the impression that that is understood, it's just that the like the "classic" style better. E.g. see https://github.com/cgeo/cgeo/discussions/11912#discussioncomment-1501405

**Reply by bekuno** [reply]
The user should recognize the icon as caching track. In the 3 point version are only points, no final. But a final is necessary for our play.
The multi boxes icon is in my opinion no good idea. Should a cacher really found multiple cache boxes?

**Reply by ztNFny** [reply]
After using the square icon for a few days again I know why I chose the 3 points over it (both icons have only 3 points btw, just in that square icon the pin takes up the room of 2 points). It looks way too busy and the square shape doesn't fit the circle around it very well. The 3 points are a lot cleaner and with the rounded lines are in harmony with the circle.

**Reply by ztNFny** [reply]
> The user should recognize the icon as caching track. In the 3 point version are only points, no final. But a final is necessary for our play. The multi boxes icon is in my opinion no good idea. Should a cacher really found multiple cache boxes?

I don't think not having one of the points marked as "final" is the problem here. Navigation apps also don't display stopovers differently than the final target.
Adding a "pin" on top of one of the points would again introduce more details and complexity into the icon, making it have the same problems that the two boxes and the current 3 version have.

**Reply by Lineflyer** [reply]
>  looks way too busy and the square shape doesn't fit the circle around it very well. The 3 points are a lot cleaner and with the rounded lines are in harmony with the circle.

@ztNFny 
There were a few comments by users here and there about "Couldn't you use a two boxes approach like the old one?".
But AFAIR you already said, that you tried but it turned out bad respectively did not look good?  Such a two-box shape would IMHO be quite close to what users know.
Otherwise: How about a three dot version but drawing the path from one to the next (instead of from left to both both points on the right)?

**Reply by ztNFny** [reply]
> How about a three dot version but drawing the path from one to the next (instead of from left to both both points on the right)?

Heh - that was already my idea and how I "read" the icon:
![image](https://user-images.githubusercontent.com/1258173/138659487-7800e638-4d41-4498-a09d-23d6ab30dbcc.png)
Idea is: S1 and S2 are visited, S3 is still open, indicated by the dotted line and the ring instead of circle.

But if you read it from left to right I see your problem. I'll try to rearrange it in a way that it goes from left to right - that'll create a whole new icon though.

> Such a two-box shape would IMHO be quite close to what users know.

Why it imho can't work I've already explained above. And going by the vote it seems that at least in the dev team a "path" approach is favored over 2 boxes. The question really comes down to "use a classical icon, no matter how good/bad it is, to avoid change" vs "create an icon that we think represents multi, fits the style of the other icons and "looks good""

**Reply by Lineflyer** [reply]
You are right...I have read it from left to right...anyway IMHO thats not the problem users have with this icons I guess.

BTW:
Is the "zoomed out becomes inrecognizable" really a problem? In fact those icons don't get zoomed out in c:geo...or are you refering to different screen resolutions?

Anyway I am still a bit undecided.... The left one in your screenshot above is fresh and new and I can identify the type by the color and meanwhile got a bit used to it...the right one is screaming: "Yes, I am the multi icon you know from c:geo, please keep me!" :)

I have one pro-argument left for the two-box icon:
For traditional we have the box, a multi cache is a sequence of traditional caches (in the very original sense at least). Thats what the icon pretty good represents.

Any chance to get more visual separation between the two boxes e.g. by taking them a few pixel further apart, so that its easier to recognize when shown smaller?

**Reply by Lineflyer** [reply]
Maybe put those options into a survey and let the beta users majority decide. If it turns out our users are more "conservative" and like the box look we go for it.

**Reply by ztNFny** [reply]
> BTW: Is the "zoomed out becomes inrecognizable" really a problem? In fact those icons don't get zoomed out in c:geo...or are you refering to different screen resolutions?

I'm referring to the normal size of the icons on the small phone screen - as opposed to the "big" size used during design, e.g. on https://ztnfny.github.io/cgeoicons/index.html (for me I have to zoom that page to ~35% to match the icon size on my phone)

> the right one is screaming: "Yes, I am the multi icon you know from c:geo, please keep me!" :)

It's shouting "I'm the one from GC" :-P

> a multi cache is a sequence of traditional caches (in the very original sense at least). Thats what the icon pretty good represents.

"Every multi has waypoints, only some have boxes/physical stages."
Looking at my find stats the traditional icon should be a petling instead of a box 🤣
 
> Any chance to get more visual separation between the two boxes e.g. by taking them a few pixel further apart, so that its easier to recognize when shown smaller?

I don't have any 2 boxes design right now (never liked any well enough to even commit them to my repo), would have to start from scratch.

> Maybe put those options into a survey and let the beta users majority decide. If it turns out our users are more "conservative" and like the box look we go for it.

The only options that actually exist right now are the 2 waypoint-styles.

**Reply by ztNFny** [reply]
Do you find this better "readable"? Quick draft only
![type_multi](https://user-images.githubusercontent.com/1258173/138948210-626d0074-eb4e-479e-a589-a6f0205d6170.png)

**Reply by Lineflyer** [reply]
IMHO yes, I have however not yet left the idea behind to keep a little bit of the c:geo two-box solution with the early draft above. But thats pure nostalgic reasoning and the fact that its just like the traditional box but taken a step further.

**Reply by bekuno** [reply]
I would also say yes. Maybe rotate it 50 to 90 deg to left to have the stroke left - top - right. And I believe that using the final marker as in the 4 point icon would make the symbol even better.

**Reply by ztNFny** [reply]
>And I believe that using the final marker as in the 4 point icon would make the symbol even better.

To use the "pin" marker from the ~~4 point~~ other 3 point icon requires the full height of the icon, then there'll only be the room on the left of it for the 2 other points - which describes the existing icon.

> I have however not yet left the idea behind to keep a little bit of the c:geo two-box solution with the early draft above.

Same problem, there isn't enough space to add a more complex shape to the icon

**Reply by ztNFny** [reply]
tried replacing the dots with waypoint markers, it'll need rearranging the positions, but I thought maybe it can work - when viewed on the large screen at 3000% size ... it's all getting way to small, the unvisited one in the upper right is pretty much invisible and even the waypoints are just a blur
![type_multi](https://user-images.githubusercontent.com/1258173/138957284-217ad7f8-1a46-4bb5-a1dd-acd40fc56102.png)

also tried putting a waypoint marker in the lower middle (pretty much the only place there's some room), doesn't work either imho
![type_multi2](https://user-images.githubusercontent.com/1258173/138957286-e5f136f8-9060-4c4e-ac8e-71d7bc2966ca.png)

If we want more than dots then the current square one and https://materialdesignicons.com/icon/map-marker-distance (which is too plain imho) are the only options I see, everything with more than that just gets too small.

**Reply by moving-bits** [reply]
I'd stay with the non-pin version: ![type_multi](https://user-images.githubusercontent.com/1258173/138948210-626d0074-eb4e-479e-a589-a6f0205d6170.png) (The "pin-version" has too much fiddly detail.)
Also would prefer this version over the original "<" shaped version.

**Reply by Lineflyer** [reply]
BTW: The Groundspeak app is using the two box solution (and the multi-color earth):
![image](https://user-images.githubusercontent.com/949669/139136677-2a0a04e4-fb8c-49cd-bbc3-3dd34c5e7d14.png)

**Reply by Lineflyer** [reply]
![image](https://user-images.githubusercontent.com/949669/139137030-4ff8995f-e0a2-4d68-a85c-f0636a18569f.png)

**Reply by ztNFny** [reply]
It's hard to make the changed order look as smooth as the old one. This is it for today...
![type_multi](https://user-images.githubusercontent.com/1258173/139148310-948c07b0-607e-436c-81b6-1c23a5433795.png)

**Reply by ztNFny** [reply]
> and the multi-color earth

They can't really tell their partner that they won't use that official icon anymore 😉

**Reply by SammysHP** [reply]
> It's hard to make the changed order look as smooth as the old one. This is it for today... ![type_multi](https://user-images.githubusercontent.com/1258173/139148310-948c07b0-607e-436c-81b6-1c23a5433795.png)

Maybe because of you interpret it as a path it doesn't make much sense. You expect to be in the foreground (at the bottom) and move away into the background (top). That's why I like the previous version.

**Reply by fm-sys** [reply]
What if we mirror the icon? Something like this... 

![Screenshot_20211028_064242](https://user-images.githubusercontent.com/64581222/139188611-5f1bd51f-d3df-49bc-ac85-cf39864bb67b.png)

**Reply by ztNFny** [reply]
> What if we mirror the icon? Something like this...

The line should curve upwards, if that is changed the spacing in that one is of. Look at the original, it's very balanced visually, something these other icons don't achieve (yet)

**Reply by ztNFny** [reply]
played around with it for a while and keep coming back to basically the original shape:
![image](https://user-images.githubusercontent.com/1258173/139486271-bf7ce1b0-0dcd-4315-b057-a40a576e51c1.png)
Any thoughts? I'm against those with waypoints as they are too busy, my favorites are type_multi_i / type_multi_i2 as the shape feels the most organic there. They start in the lower left, going to middle right, upper left. For me that "feels" ok, as I read it like a map where you also start at the bottom. Start at top left like i5 isn't as smooth imho.

**Reply by SammysHP** [reply]
`type_multi_i.svg` or `type_multi.svg`.

**Reply by ztNFny** [reply]
Took another stab at a 2 box version Shouldn't surprise anyone at this point that it's not my favorite, but as a backup, if the voices against waypoints are too strong, it's okish.
![type_multi_2box](https://user-images.githubusercontent.com/1258173/139492043-f59fdfaf-f89a-45d4-b0b0-a12c60d35c58.png)
![type_multi_2box](https://user-images.githubusercontent.com/1258173/139492415-507b9ad8-5c87-4846-ac93-907098c18810.png)
![image](https://user-images.githubusercontent.com/1258173/139492480-3ccc859d-83b5-4759-9039-0c312b7f1ae8.png)

**Reply by bekuno** [reply]
Favorites are `i3` before `i`.

To not duplicate the geocaching.com icon (in `2box`), how about using a pin marker instead the rear box?

**Reply by fm-sys** [reply]
>how about using a pin marker instead the rear box?

Good idea. We could use the GC two boxes style, but with pin markers instead. Something like this (maybe with only two markers):

![Screenshot_20211030_150904_org mozilla firefox_copy_91x108](https://user-images.githubusercontent.com/64581222/139534096-0cb471bd-4276-4804-a50d-d0dadbc55647.png)

**Reply by SammysHP** [reply]
Honestly I wouldn't understand the meaning of one or more stacked pins.

**Reply by moving-bits** [reply]
Favorites are `multi_i` > `multi_i2` > `multi` > `multi_it`

**Reply by murggel** [reply]
My favorite is one of multi_i, multi_i2 or multi_i3

**Reply by ztNFny** [reply]
multi_i seems to be the favorite, so let's go with that. And then we can decide with the input from the survey whether we use this or the boxes version.

**Comment by Lineflyer** [comment]
One more comment from Facebook (c:geo page). There have been a handful with the topic "white circle around", here is one example:

In my opinion, this was a good oveall change with some room for improvement. I would remove the white circle or change it to a thinner version.

**Reply by ztNFny** [reply]
The white circle (and its black border) is required to provide some contrast against the map and other caches. Without it everything quickly turns into a big blob:
![image](https://user-images.githubusercontent.com/1258173/137886847-16c970c6-3cf8-4762-aede-0e830c6f7537.png)
![image](https://user-images.githubusercontent.com/1258173/137907159-60f8f7a2-4942-4eb4-acb3-c088ae9634cf.png)

![image](https://user-images.githubusercontent.com/1258173/137887027-e0125502-62f7-4427-99b6-ade194b432d0.png)
![image](https://user-images.githubusercontent.com/1258173/137906781-7e7cde41-25ae-4ec7-8b6a-879440a336e5.png)


It's a design element too, of course. So some will probably like it better, some without, but keeping accessibility in mind I think removing it would be a loss.

**Reply by murggel** [reply]
I would not remove it, but probably a little bit thinner. Don't know, if the proportion than still looks good...

**Reply by ztNFny** [reply]
The "ring" size is aligned to the pin size, the background size is aligned with the icons. Changing one means also having to change all the others.
Here's the ring with half the width:
![image](https://user-images.githubusercontent.com/1258173/137905293-d417efab-ee5b-4465-b7bc-e22c3950e3df.png)
Here's one with a slightly reduced ring (23.4dp instead of 22.4dp for the background circle), that kinda is the maximum possible without having to touch everything again:
![image](https://user-images.githubusercontent.com/1258173/137905828-e5daab91-929b-4f87-b6b5-010b73b0a6a9.png) 
![image](https://user-images.githubusercontent.com/1258173/137906548-de2189cf-5c20-4635-a316-340e9dd071c1.png)

**Reply by murggel** [reply]
Thanks! 
Personally I like the second one better.

**Reply by ztNFny** [reply]
23.4dp implemented

**Comment by ztNFny** [comment]
> Multi in Orange mit Stationen finde ich persönlich auch nicht ok, jeder Cacher weiß, was ein Multi ist.

Source: Facebook

**Reply by ztNFny** [reply]
So is the complaint that the multi icon looks too intuitive?? 🤣
